waywiser_command_control/launch/command_control.launch.py

```diff
@@ -32,7 +32,7 @@ def generate_launch_description():
                 'use_sim_time': LaunchConfiguration('use_sim_time'),
             },
         ],
-        remappings={('/cmd_vel_in', '/onboard_mux_vel')},
+        remappings={('/cmd_vel_in', '/cmd_vel_safe')},
```

Comment:
It would be great to have a node graph as part of the documentation of waywiser_command_control. It is IMO quite hard to follow how the nodes are connected and which node does what, this should also be updated in the package.xml. Is it now mandatory to run the whole set of nodes or can you, e.g., run emergency stop only?

I would prefer naming cmd_vel_safe s.th. like cmd_vel_collisionmon as cmd_vel_safe suggests it is safe already (why should it go to emergency stop then?).

Reply:
Updated the readme file and package.xml. It is not mandatory to run all nodes in command_control launch file. Emergency stop node can be run alone, but its probably better to have another launch file for this case. Nonetheless, launching all the nodes along with the collision monitor would still work even if the RC car does not have any lidar or depth camera. Renamed cmd_vel_safe to collision_monitor_vel.